src/pages/CreatePool/index.tsx

```diff
@@ -101,9 +122,16 @@ export default function CreatePool(): React.ReactElement {
                   label={`${t("fee")} (%)`}
                   fullWidth
                   value={fee}
-                  error={!digitRegex}
+                  error={
+                    !digitRegex || parseFloat(fee) > 1 || parseFloat(fee) < 0.04
```

Comment:
This is used twice. It would be nice to store reusable logic like this in a `const`.

Reply:
Have you tried using this suggestion to prevent the user from typing non-digits completely?

https://mui.com/components/text-fields/#type-quot-number-quot

Reply:
Great question!
Yeah, I have tried this approach when I implement the `SwapTokenInput` and it popup message when user click `submit` button like the follwoing.
![image](https://user-images.githubusercontent.com/75422800/159909981-65e43f99-3dab-4f83-9619-2ab581752351.png)

In our case, we need to notify user when user input wrong value, so I think it would be good to use this approach with current logic.

Reply:
current logic is fine